docker/templates/debian/runtime.Dockerfile.j2

```diff
@@ -7,10 +7,8 @@ FROM {{ metadata.package }}:{{ build_tag }} as build_image
 
 ENV BENTOML_VERSION={{ metadata.envars['BENTOML_VERSION'] }}
 
-RUN pip install bentoml[{{ metadata.package | replace("-","_") }}]==${BENTOML_VERSION} {{ pip_options|join(' ') }}
+RUN pip install bentoml[model_server]==${BENTOML_VERSION} {{ pip_options|join(' ') }}
 
 COPY tools/{{ metadata.package }}/entrypoint.sh /usr/local/bin/
 
-ENTRYPOINT [ "entrypoint.sh" ]
-
-CMD ["bentoml", "serve-gunicorn", "/bento"]
\ No newline at end of file
+ENTRYPOINT [ "entrypoint.sh" ]
```

Comment:
also remove the entrypoint here? same as CMD, it will be added at the last layer

Reply:
:( will do